### ISSUE TITLE
`gradle -I ... init` returns error `Could not open cp_init generic class cache for initialization script`

### ISSUE THREAD

**Issue by Zhxhh** [NationalSecurityAgency/ghidra#3234]
```
ghidra-Ghidra_9.2.2_build %  gradle -I ./gradle/support/fetchDependencies.gradle init

FAILURE: Build failed with an exception.

* What went wrong:
Could not open cp_init generic class cache for initialization script '/Users/bytedance/Downloads/ghidra-Ghidra_9.2.2_build/gradle/support/fetchDependencies.gradle' (/Users/bytedance/.gradle/caches/6.9/scripts/4avd8tbmgkox3vb6vswpmq1lo).
> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 60

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 472ms
```
I'm using macOS15.5
```
% gradle -version

------------------------------------------------------------
Gradle 6.9
------------------------------------------------------------

Build time:   2021-05-07 07:28:53 UTC
Revision:     afe2e24ababc7b0213ccffff44970aa18035fc0e

Kotlin:       1.4.20
Groovy:       2.5.12
Ant:          Apache Ant(TM) version 1.10.9 compiled on September 27 2020
JVM:          16.0.1 (Oracle Corporation 16.0.1+9-24)
OS:           Mac OS X 10.15.5 x86_64
```

**Comment by ryanmkurtz** [comment]
I don't think Gradle 6.9 supports running with Java 16.  You'll need to upgrade to Gradle 7 for that.  However, only Ghidra 10.x supports Gradle 7, so you'll have to upgrade your Ghidra if you want to continue using Java 16.  Alternatively, you could use JDK 11 and keep your Gradle and Ghidra the same.

**Comment by astrelsky** [comment]
@ryanmkurtz doesn't Ghidra have problems with jdk 16 due to the illegal reflection access in cglib that is no longer allowed?

**Comment by ryanmkurtz** [comment]
> @ryanmkurtz doesn't Ghidra have problems with jdk 16 due to the illegal reflection access in cglib that is no longer allowed?

That was fixed in 9.2.4.

**Comment by astrelsky** [comment]
> > @ryanmkurtz doesn't Ghidra have problems with jdk 16 due to the illegal reflection access in cglib that is no longer allowed?
> 
> That was fixed in 9.2.4.

Oh, I hadn't noticed :rofl:
